openhands/llm/metrics.py

```diff
@@ -55,11 +55,9 @@ def add_cost(self, value: float) -> None:
         self._costs.append(Cost(cost=value, model=self.model_name))
 
     def add_response_latency(self, value: float, response_id: str) -> None:
-        if value < 0:
-            raise ValueError('Response latency cannot be negative.')
```

Comment:
My first thought was to downgrade to `logger.warning`, but I was getting some very strange behavior in unit tests when I used the default `openhands_logger` implementation.

Reply:
I think this solution is fine for general use, it may be better than a lot of warnings after all. We could maybe even make it negative, e.g. -1, just to be able to see at a glance (and easily avoid it in calculations). But 0 works too IMHO.